oif/helpers.py

```diff
@@ -24,8 +25,30 @@
 
 @cache
 def timestamp_of_block(block_number: int) -> int:
+  """ Returns the timestamp of block :attr:`block_number` """
   return web3.eth.get_block(block_number)['timestamp']
 
+@cache
+def block_of_timestamp(target_timestamp: int) -> int:
+  """ Returns the block number that is closest in time to the :attr:`target_timestamp` """
+  avg_block_time = 15
```

Comment:
I think this would better fit in a `constants.py` file (I used a 13.6 number I got from an Ethereum statistics website).